tox.ini

```diff
@@ -4,7 +4,7 @@ requires =
     tox-run-command
  
  
-[testenv:py{39,310,311}-lock]
+[testenv:py{310,311,312}-lock]
```

Comment:
There are some missing corresponding changes. This is creating some confusing conflicts with #423. I could raise a pull request to fix these if you're OK with that?

https://github.com/SciTools/cf-units/blob/8a82027f25c97cfe531ff68e1b0ec09b1c0b2fa0/setup.cfg#L8-L12

https://github.com/SciTools/cf-units/blob/8a82027f25c97cfe531ff68e1b0ec09b1c0b2fa0/setup.cfg#L45-L46

https://github.com/SciTools/cf-units/blob/8a82027f25c97cfe531ff68e1b0ec09b1c0b2fa0/.github/workflows/ci-locks.yml#L28-L30

https://github.com/SciTools/cf-units/blob/8a82027f25c97cfe531ff68e1b0ec09b1c0b2fa0/.github/workflows/ci-tests.yml#L33-L42

https://github.com/SciTools/cf-units/blob/8a82027f25c97cfe531ff68e1b0ec09b1c0b2fa0/pyproject.toml#L50

https://github.com/SciTools/cf-units/blob/8a82027f25c97cfe531ff68e1b0ec09b1c0b2fa0/tox.ini#L36-L45